tests/test_in_toto_verify.py

```diff
@@ -572,6 +156,37 @@ def test_main_multiple_keys(self):
         ] + self.public_key_paths
         self.assert_cli_sys_exit(args, 0)
 
+    def test_main_failing_bad_layout_path(self):
+        """Test in-toto-verify CLI tool with bad layout path."""
+        args = [
+            "--layout",
+            "not-a-path-to-a-layout",
+            "--verification-keys",
+        ] + self.public_key_paths
+        self.assert_cli_sys_exit(args, 1)
+
+    def test_main_link_dir(self):
+        """Test in-toto-verify CLI tool with explicit link dir."""
+        # Use current working directory explicitly to load links
+        args = [
+            "--layout",
+            "demo.layout",
+            "--link-dir",
+            ".",
+            "--verification-keys",
+        ] + self.public_key_paths
+        self.assert_cli_sys_exit(args, 0)
+
+        # Fail with an explicit link directory, where no links are found
+        args = [
+            "--layout",
+            "demo.layout",
+            "--link-dir",
+            "bad-link-dir",
+            "--verification-keys",
+        ] + self.public_key_paths
+        self.assert_cli_sys_exit(args, 1)
+
 
 class TestInTotoVerifySubjectPublicKeyInfoKeysAndUseDSSE(
```

Comment:
Given our "defaults" have effectively changed, maybe this test could be renamed. I was a little thrown by the removal of `TestInTotoVerifyTool` above. But also, it may not be an issue once this PR lands and it's a lot clearer what the tests are doing in the file...

Reply:
🤷 I don't like the name either. But I don't mind it enough to rename.